src/local-mode.ts

```diff
@@ -40,9 +40,20 @@ export class LocalModeManager {
 			}
 		});
 
-		const localMode = checkTruthy(
-			(await this.config.get('localMode')) || false,
-		);
+		// On startup, check if we're in unmanaged mode,
+		// as local mode needs to be set
+		let unmanagedLocalMode = false;
+		if (checkTruthy((await this.config.get('unmanaged')) || false)) {
+			console.log('Starting up in unmanaged mode, activating local mode');
+			this.config.set({ localMode: true });
```

Comment:
You okay with doing this in the background? Or should this have an `await`?

Reply:
Also, now that this is a db value, should this be a string? i.e. `'true'`?

Reply:
Good catch on the `await`, will change. I'd much rather handle the conversion to string on the way into the database though, as that's an implementation detail, and shouldn't be part of the interface.